includes/Experiments.php

```diff
@@ -219,6 +219,17 @@ public function get_experiment_groups() {
 	 */
 	public function get_experiments() {
 		return [
+			/**
+			 * Author: @littlemilkstudio
+			 * Issue: 7965
+			 * Creation date: 2021-06-18
+			 */
+			[
+				'name'        => 'enableChecklistCompanion',
+				'label'       => __( 'Enable Checklist Companion', 'web-stories' ),
+				'description' => __( 'Enables v2 of prepublish checklist as a popup in the editor.', 'web-stories' ),
```

Comment:
```suggestion
				'description' => __( 'Enable v2 of prepublish checklist as a popup in the editor.', 'web-stories' ),
```

nit: making the language consistent with the other experiments.